admin/src/commands/lint.rs

```diff
@@ -17,6 +17,9 @@ pub struct LintCmd {
         help = "filesystem path to the RustSec advisory DB git repo"
     )]
     path: Vec<PathBuf>,
+
+    #[clap(long, help = "Skip name check comma separated crates list")]
+    skip_namecheck: Option<String>,
```

Comment:
`clap`/`structopt` at least used to have first-class support for comma-separated arguments, although I'm not sure what the status is with `clap` v3 (the `clap_derive` docs are 404ing, unfortunately)